packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/VirtualizedTraceView.tsx

```diff
@@ -55,9 +48,9 @@ type TVirtualizedTraceViewOwnProps = {
   scrollToFirstVisibleSpan: () => void;
   registerAccessors: (accesors: Accessors) => void;
   trace: Trace;
-};
+  focusSpan: (uiFind: string) => void;
 
-type TDispatchProps = {
+  // was from redux
```

Comment:
rather than commenting this, a type should be imported and `TVirtualizedTraceViewOwnProps` can be called `TProps` (no separation of own vs dispatch props) which would look like:
```
TProps = {
  currentViewRangeTime: [number, number];
  findMatchesIDs: Set<string> | TNil;
  focusSpan: (uiFind: string) => void;
  scrollToFirstVisibleSpan: () => void;
  registerAccessors: (accesors: Accessors) => void;
  trace: Trace;
} & TImportedType;
```